changes/5326.fixed

```diff
@@ -0,0 +1 @@
+Fixed simple typo in documentation.
```

Comment:
Thank you for the contribution @edwardtheharris! would you mind changing the file name from `.fixed` to `.housekeeping?`

Reply:
Should be `.documentation` shouldn't it?

Reply:
Ahh I was not sure about the difference between **fixing** the documentation and **adding** documentation. But it makes sense that both would go under .`documentation`.👍